roles/cinder-data/tasks/monitoring.yml

```diff
@@ -7,3 +7,9 @@
 - name: iscsid process check
   sensu_process_check: service=iscsid
   notify: restart sensu-client
+
+- name: cinder volume group metrics
+  sensu_metrics_check: name=cinder-vg-metrics plugin=metrics-volgroup.sh
+                       args='-v cinder-volumes --scheme {{ monitoring.graphite.host_prefix }}'
+  notify: restart sensu-client
+  when: cinder.fixed_key is not defined or (cinder.fixed_key is defined and "compute" not in group_names)
```

Comment:
we don't allow cinder/LVM  when: 'cinder.fixed_key is defined and "compute" not in group_names' therefore we shouldn't do the monitoring for it.